### ISSUE TITLE
error generate error

### ISSUE THREAD

**Issue by OfficialTooFastHub** [abi/screenshot-to-code#407]
**Describe the bug**
everything work well and looks well but it came out error generating code while i drop a screenshot

**To Reproduce**
1. Go to ['local host '](http://localhost:5173/)
2. Drag the screenshot
3. Start Generating

**Screenshots of backend AND frontend terminal logs**
![Screenshot 2024-09-26 234546](https://github.com/user-attachments/assets/26840dc7-f680-41ec-b6b9-6594836dc82e)
![Screenshot 2024-09-26 235009](https://github.com/user-attachments/assets/b0a233ba-07f7-421c-9302-dcd900a59362)
![Screenshot 2024-09-26 235052](https://github.com/user-attachments/assets/3473a4ee-9e87-4cc1-a2cd-2e1bb1650f8b)

**Comment by abi** [comment]
Thanks for the screenshots. Have you added a Anthropic API key?

<img width="802" alt="Screenshot 2024-09-30 at 11 59 23 AM" src="https://github.com/user-attachments/assets/42c8ae5b-a4b7-49d8-a09c-db6c826ac75d">

But maybe your logs include multiple attempts. Looks like maybe even just screenshot without video is not working? 

Can you let me know which keys you have set up? It's a little weird that we're not seeing any exception stack traces in your logs.

**Comment by OfficialTooFastHub** [comment]
I just added chat gpt API key, didn't add Anthropic API key,does it require two keys to work?

**Comment by OfficialTooFastHub** [comment]
![Screenshot 2024-10-03 164619](https://github.com/user-attachments/assets/51c90a4f-5099-4b42-80a7-be53f7d83eeb)
still same,i clear out anthropic and kepp openai keys,but it still not working

**Comment by abi** [comment]
Hi, can you share your backend logs after you did this?

**Comment by OfficialTooFastHub** [comment]
![Screenshot 2024-10-06 002627](https://github.com/user-attachments/assets/fbd6159c-f660-4df6-8d76-bea08d2c9c66)
![Screenshot 2024-10-06 002615](https://github.com/user-attachments/assets/44aa1dc8-3bc1-43c6-ad40-9037ef20ad0b)
here is it,i think there is some error

**Comment by abi** [comment]
Hi @OfficialTooFastHub this looks to be a different error. 

Try downgrading your Python to python:3.12.3 or lower. The new version of Pydantic has some issues with the later Python versions.

Let me know if that works.

**Comment by OfficialTooFastHub** [comment]
@abi yo i download python 3.12.3,but the error still coming out
![Screenshot 2024-10-08 211303](https://github.com/user-attachments/assets/59294416-f181-4cc3-8c5d-249cbc2d572a)
![Screenshot 2024-10-08 211253](https://github.com/user-attachments/assets/64e672ca-97c3-4a74-af88-4025fbfd9c3e)

**Comment by abi** [comment]
If you look at the second error, you can see that your API key is wrong.

**Comment by OfficialTooFastHub** [comment]
ill try again

**Comment by OfficialTooFastHub** [comment]
should i keep the "" in the key section

**Comment by abi** [comment]
@OfficialTooFastHub Nope, but I'm not sure that's the problem. It seems to be reading it without the quotes.

<img width="251" alt="Screenshot 2024-10-08 at 10 03 04 AM" src="https://github.com/user-attachments/assets/0ceb7322-a0a8-4c42-a6d3-716629e3b18f">
